app/src/main/java/com/sopt/now/compose/SoptApplication.kt

```diff
@@ -0,0 +1,11 @@
+package com.sopt.now.compose
+
+import android.app.Application
+import dagger.hilt.android.HiltAndroidApp
+
+@HiltAndroidApp
+class SoptApplication : Application() {
+    override fun onCreate() {
+        super.onCreate()
+    }
+}
```

Comment:
```suggestion
@HiltAndroidApp
class SoptApplication : Application()
```

내부에서 override하는게 없기때문에 이렇게만 써도 될거에요~
물론 추후에 onCreate에서 뭔가를 해야한다면 써야해용!!